XCUITests/SearchTest.swift

```diff
@@ -218,4 +218,38 @@ class SearchTests: BaseTestCase {
         waitForExistence(app.textFields["url"], timeout: 20)
         waitForValueContains(app.textFields["url"], value: "google")
     }
+    
+    func testSearchIconOnAboutHome() {
+        waitForTabsButton()
+        navigator.performAction(Action.OpenNewTabFromTabTray)
+        waitForTabsButton()
```

Comment:
No need to open a new tab to verify the search button. It is displayed right away when the app is open:
<img width="311" alt="Captura de pantalla 2020-08-12 a las 11 42 57" src="https://user-images.githubusercontent.com/1897507/90000947-18a42280-dc91-11ea-9619-5b46603bb4bd.png">

I would start the test with for example: `waitForExistence(app.cells["TopSitesCell"])` to be sure the app is displayed and from there, tap on the Search button

Reply:
Ok, so you don't just wait for the icon itself?  Is this approach better that you're indicating?

Reply:
That's a good approach to, wait for that buttom, any icon in the screen to be sure the screen is ready and the icon should be there already. But on previous lines, that was not the case yet, that's why I suggested that :)

Reply:
Just doubled checked with Simon and the test defined in TR, and yeah we could leave these steps to open a new tab. But I will also check that starting the app the search button is present. 
So 
```
testXxxxYyyyyy(){
        // Search icon is displayed.
        waitForExistence(app.buttons["TabToolbar.stopReloadButton"])
        XCTAssertEqual(app.buttons["TabToolbar.stopReloadButton"].label, "Search")
        XCTAssertTrue(app.buttons["TabToolbar.stopReloadButton"].exists)
       
        // Open a new tab
        navigator.performAction(Action.OpenNewTabFromTabTray)
        
        // Search icon is displayed.
        waitForExistence(app.buttons["TabToolbar.stopReloadButton"])
        XCTAssertEqual(app.buttons["TabToolbar.stopReloadButton"].label, "Search")
        XCTAssertTrue(app.buttons["TabToolbar.stopReloadButton"].exists)
        
        .......Rest of the test
}
```